Dockerfile

```diff
@@ -3,10 +3,15 @@ FROM node:12.18.0-alpine3.12
 RUN apk add --no-cache sudo shfmt
 RUN npm install --global --silent yo
 
+# copy fabrica files
 COPY generators /fabrica/generators
 COPY package.json /fabrica/package.json
 COPY package-lock.json /fabrica/package-lock.json
 
+# copy files for init network
+COPY samples/fabricaConfig-1org-1channel-1chaincode.json /fabrica/generators/init/templates/fabrica-config.json
+COPY samples/chaincodes/chaincode-kv-node /fabrica/generators/init/templates/chaincodes/chaincode-kv-node
+
 WORKDIR /fabrica
 RUN npm install --silent
```

Comment:
Probably not for this PR, but I've just noticed it - we copy `generators /fabrica/generators` invoke `npm install` quite early. And they are frequently changed.  
If we postpone it to later we could save some image build time thanks to docker cache.

Reply:
we decided not to change it